pkg/simpleschema/transform.go

```diff
@@ -275,6 +275,25 @@ func (tf *transformer) applyMarkers(schema *extv1.JSONSchemaProps, markers []*Ma
 				},
 			}
 			schema.XValidations = validation
+		case MarkerTypeImmutable:
+			switch isImmutable, err := strconv.ParseBool(marker.Value); {
+			case err != nil:
+				return fmt.Errorf("failed to parse immutable marker value: %w", err)
+			case isImmutable:
```

Comment:
i would just use an if/else here
```suggestion
			isImmutable, err := strconv.ParseBool(marker.Value); {
			if err != nil{
				return fmt.Errorf("failed to parse immutable marker value: %w", err)
		    }
		    schema.XValidations = append(schema.XValidations, immutableValidation...)
```